index.js

```diff
@@ -41,6 +41,7 @@ function portToUrlString(options) {
  *   @param    {Hash}       options
  *   @param    {Boolean}    options[trustProtoHeader]
  *   @param    {Boolean}    options[trustAzureHeader]
+ *   @param    {String}     options[customProtoHeader]
```

Comment:
it would be great if you could also add some note about this to the README as it's, in fact, the only kind of documentation at this point. But I'll be able to do it myself as well so it's not really a blocker.

Reply:
done